x-pack/solutions/security/plugins/security_solution/public/data_view_manager/hooks/use_selected_patterns.ts

```diff
@@ -7,10 +7,10 @@
 
 import { useMemo } from 'react';
 import type { DataViewManagerScopeName } from '../constants';
-import { useDataView } from './use_data_view';
+import { useDataViewSpec } from './use_data_view_spec';
 
 export const useSelectedPatterns = (scope: DataViewManagerScopeName): string[] => {
-  const { dataView } = useDataView(scope);
+  const { dataViewSpec: dataView } = useDataViewSpec(scope);
```

Comment:
while in some files I understand that you'd want to do `dataViewSpec: dataView` to avoid having to rename too many places, here for example (and in other places in this PR) I don't really see the value.
It makes the code more complex and harder to read... There are only 2 instances of the word `dataView` right below that can easily be changed too?